integration-cli/docker_cli_swarm_unix_test.go

```diff
@@ -50,3 +51,56 @@ func (s *DockerSwarmSuite) TestSwarmVolumePlugin(c *check.C) {
 	c.Assert(mounts[0].Name, checker.Equals, "my-volume")
 	c.Assert(mounts[0].Driver, checker.Equals, "customvolumedriver")
 }
+
+// Test network plugin filter in swarm
+func (s *DockerSwarmSuite) TestSwarmNetworkPluginV2(c *check.C) {
+	d1 := s.AddDaemon(c, true, true)
+	d2 := s.AddDaemon(c, true, false)
+
+	// install plugin on d1 and d2
+	pluginName := "aragunathan/g3:latest"
```

Comment:
@AkihiroSuda : Plugin source is at https://github.com/mavenugo/test-docker-netplugin/
The changes that were required to make it a globally scoped network plugin are in this PR https://github.com/mavenugo/test-docker-netplugin/pull/1